app/pulp/app/settings.py

```diff
@@ -170,13 +170,9 @@
             },
         },
         'loggers': {
-            'pulp.app': {
-                'handlers': ['stream'],
-                'level': 'INFO',
-            },
-            'django': {
-                'handlers': ['stream'],
-                'level': 'INFO',
+            '': {
+                # The root logger
+                'handlers': ["stream"],
```

Comment:
These changes need to be also updated in the server.yaml file itself.  https://github.com/pulp/pulp/blob/3.0-dev/app/etc/pulp/server.yaml#L38